test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -349,3 +352,148 @@
                          :else (do
                                  (Thread/sleep 500)
                                  (recur (dec times)))))))))))
+
+;; This may need to be moved to one of the utils file/ns
+(defn get-ca-signed-cert
+  [ca-cert
+   certname
+   serial
+   master-conf-dir]
+  (let [ca-public-key  (.getPublicKey ca-cert)
+        ca-private-key (ssl-utils/pem->private-key
+                        (str master-conf-dir "/ssl/ca/ca_key.pem"))
```

Comment:
We load the private-key based on the confdir but have the cert passed in, why is that? Should we either have the caller pass in the CA objects or load them from the path relative to confdir?